scripts/win32.bat

```diff
@@ -39,6 +40,7 @@ For Each objDrive In colDiskDrives
                 objPartition.DeviceID & """} WHERE AssocClass = " & _
                     "Win32_LogicalDiskToPartition")
         For Each objLogicalDisk In colLogicalDisks
+            foundLDs = foundLDs + 1
```

Comment:
Given that we only care if at least one logical disk was found, and not how many, I recommend this variable to be a boolean one instead, called `containsLogicalDisk` or something like that.

If we reach to this point, we can set to `True`, and then compare with `False` below.